pkg/controller/utils/url_validation.go

```diff
@@ -0,0 +1,97 @@
+package utils
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// IsValidJWTIssuerURL validates JWT issuer URL format following SPIRE patterns
```

Comment:
nit: As the URL validation is not specific to JWT, can it be renamed to IsValidURL and remove references to "JWT" in the url_validation.go file?